modules/mobile/spec/validations/translations_validation_spec.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
```

Comment:
It would have been nice to only run these specs when the file has been changed and perhaps we can do that in the future. It wouldn't be particularly difficult to check see if the file was changed in the current sha. But I couldn't find any similar tests anywhere and decided not to do anything out of the ordinary. These tests only take about .25 seconds to run locally, so they're not adding too much overhead. Hopefully we'll move the translations to AWS soon anyhow.